app/ts/library/exceptions.ts

```diff
@@ -0,0 +1,23 @@
+export type MetaMaskRpcErrorSignature = {
```

Comment:
This is called `EthereumJsonRpcError` in Interceptor. As it's not specific to Metamask only

Reply:
The error I'm trying to capture actually only happens on MetaMask, thus I tried to capture the signature/shape to try and distinguish it. We may add a more general Ethereum Error capture later, at the moment only trying these out with MetaMask and Brave built-in wallets and figure a mechanism later when introducing broader support